Resources/views/Collector/migrations.html.twig

```diff
@@ -13,29 +11,34 @@
             {% set status_color = new_migrations > 0 ? 'red' : status_color %}
 
             {% set icon %}
-                {{ include('@DoctrineMigrations/Collector/icon.svg') }}
+                {% if profiler_markup_version == 3 %}
```

Comment:
this should use `>= 3`. If a v4 redesign is done, it is better for the bundle to show the v3 design than the v2 one when it is not updated yet.

Reply:
Another alternative is to use a `< 3` condition as you did in DoctrineBundle, by reversing the branches

Reply:
@javiereguiluz any chance to apply the request change ?

Reply:
I've just updated this. I'm sorry for doing it so late 🙏

Reply:
I still don't see it as updated. Did you forgot to push your change ?

Reply:
Yes, sorry 😞  It should be fixed now. Thanks.

Reply:
@javiereguiluz I still have this `{% if profiler_markup_version == 3 %}` in the diff of the PR.

Reply:
How strange. IS this a GitHub issue or am I doing something wrong and I don't realize?

The 4th commit of this PR is this one: https://github.com/doctrine/DoctrineMigrationsBundle/pull/480/commits/4eaeea2eeedca8a7d34454d871ac568ea6daae30

Reply:
this commit changes line 69. This comment is on line 14.

Reply:
OK, now I feel like an idiot 🤦  I've just updated the line 14 too.